Detections/Corelight/CorelightMultipleFilesSentOverHTTPAbnormalRequests.yaml

```diff
@@ -0,0 +1,34 @@
+id: 7226d37b-50ee-4e3b-9f80-5b74080d8f2c
+name: Corelight - Multiple files sent over HTTP with abnormal requests
+description: |
+  'Detects sources sending multiple compressed files greater than 10MBs sent over HTTP in a short amount of time.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated